pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -1,5 +1,7 @@
 { stdenv, nixosTests, lib, edk2, util-linux, nasm, acpica-tools, llvmPackages
 , csmSupport ? false, seabios ? null
+, fdSize2MB ? csmSupport
```

Comment:
Not sure why fdSize2MB defaults on csmSupport

Reply:
Ah yes I see

Reply:
I didn't dig into why it was already this way, but yeah this keeps the existing output the same for CSM.